From f5f1a4821dcea9b515abb35a9fb01e7909ee6664 Mon Sep 17 00:00:00 2001
From: Franck LENORMAND <franck.lenormand@nxp.com>
Date: Thu, 20 Sep 2018 10:16:41 +0200
Subject: [PATCH 4726/5242] MLK-19365: Revert "crypto: testmgr - Disable
 rfc4543 test"

commit  435dd96090410a7aa6e965aef4a19f973a8044ba from
https://source.codeaurora.org/external/imx/linux-imx.git

The reverted commits was disabling some code because it was
not compatible. Now it is.

This reverts commit 2570172aabd1962b953625283587541424f7b6a4.

Signed-off-by: Franck LENORMAND <franck.lenormand@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 crypto/testmgr.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/testmgr.c b/crypto/testmgr.c
index 3f8e037..8e5f1cf 100644
--- a/crypto/testmgr.c
+++ b/crypto/testmgr.c
@@ -3324,7 +3324,7 @@ static int alg_test_null(const struct alg_test_desc *desc,
 			}
 		}
 	}, {
-		.alg = "rfc4543(gcm(aes))-disabled",
+		.alg = "rfc4543(gcm(aes))",
 		.test = alg_test_aead,
 		.suite = {
 			.aead = {
-- 
1.7.9.5

